h3ronpy/tests/pandas/test_vector.py

```diff
@@ -101,6 +57,9 @@ def test_empty_geometrycollection_omitted():
     assert len(df) == 0
 
 
+@pytest.mark.skip(
+    reason="Empty points are unsupported until https://github.com/geoarrow/geoarrow-rs/issues/852 is fixed"
```

Comment:
Thanks for the reference to the issue.

I suppose you prefer not to change the API to return a Result which can be handled? It would be a bit of a special case as the whole conversion process is infallible

Reply:
In https://github.com/georust/geo/pull/1255 we propose having both panicking and non-panicking versions. The non-panicking version returns an `Option<geo::Point>`. Feel free to comment on that issue if you'd like.

The primary problem is that simple features (and geo-traits) defines empty points, but `geo` does not support them.